ambry-store/src/main/java/com/github/ambry/store/DiskManager.java

```diff
@@ -371,6 +375,8 @@ boolean addBlobStore(ReplicaId replica) {
         // create a bootstrap-in-progress file to distinguish it from regular stores (the file will be checked during
         // BOOTSTRAP -> STANDBY transition)
         createBootstrapFileIfAbsent(replica);
+        // Update the disk used capacity
+        disk.addUsedCapacityInBytes(replica.getCapacityInBytes());
```

Comment:
I only saw the disk's used capacity increases when adding blob store, but we should have logic when the disk used capacity is decreased due to removing replica.

Reply:
Thank you for catching it. Fixed it.